### ISSUE TITLE
Issues surrounding typing in the query editor window while results returning

### ISSUE THREAD

**Issue by gnarlynietzsche** [sqlpad/sqlpad#1113]
If I execute a query that takes longer than a couple of seconds to return my results, and I begin typing on a new line while the query is executing I lose the ability to type.

Steps:

1. Execute a long running query, or one that will reliably take enough time for you to be able to start typing in the query editor window
2. While the query is running, hit return return > enter whatever text you would like
3. When the results are rendered in the results grid, I lose the ability to type. The solution is to swap to a new tab.

This happens across the board on my team. We are a team of web devs and back end engineers. We have tried a myriad of settings in chrome and tried other browsers and we are not able to figure out what is going on. We can reliably reproduce this. Swapping to a new tab in the browser always resolves this issue and we are then able to begin typing again. This issue has plagued us for several years, or as long as I have worked with it (4 years). 

Let me know if you need anything for me, or if you have any ideas or items that you can think to try to fix the issue.

**Comment by rickbergfalk** [comment]
Agh! I feel your pain @gnarlynietzsche . I thought this was addressed. The best I could tell was its some quirk between the Ace editor used for editing, and the window virtualization done for the results, the the shift of focus between the two.

What version of SQLPad are you running? Does this still occur for you on latest? (I was unable to reproduce)

**Comment by rickbergfalk** [comment]
At one point there was a hacky solution to this, but at some point it was no longer a problem and the fix was removed

https://github.com/sqlpad/sqlpad/blob/141c87b8e24738263e5cd0a5ed584b1b59fe0aed/client/src/common/QueryResultDataTable.js#L227

**Comment by gnarlynietzsche** [comment]
Thanks Rick! I actually ended up discussing this with one of our Director's at work after I submitted it. I imagined someone would ask about the versioning, so I reached out to get the details. They mentioned that we are running an older version, but didn't have the exact version number. So, this can probably be discarded. I will look internally about updating and if the issue persists, I can reach back out. However, as your replies indicate, I would imagine that if we were to update our tool it wont likely be an issue. Thank you for taking the time to reply, and especially for confirming that I was not insane for noticing this. I am not sure how you handle these submissions, but as I said, probably best to just discard it or I will close it if thats all that needs to be done.
